task1Daniv/stringlist.cpp

```diff
@@ -0,0 +1,250 @@
+#include "stringlist.h"
+
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
Used restricted header **stdio.h**

Reply:
I agree with Nastia' comments. Also comment out your code, describe what you do and why (Reading code is 80% of the programming process).